src/app.rs

```diff
@@ -176,76 +152,86 @@ impl<Ms, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> App<Ms, Mdl, INodes
+                popstate_closure: RefCell::new(None),
+                hashchange_closure: RefCell::new(None),
+                window_event_handler_manager: RefCell::new(EventHandlerManager::new()),
+                // @TODO_B: We are trying to get rid of using SubManager
+                sub_manager: RefCell::new(crate::app::sub_manager::SubManager::new()),
+                msg_listeners: RefCell::new(Vec::new()),
+                scheduled_render_handle: RefCell::new(None),
+                after_next_render_callbacks: RefCell::new(Vec::new()),
+                render_info: Cell::new(None),
+            }),
         };
-        let app = Self::new(
-            update,
-            None,
-            view,
-            root_element,
-            None,
-            None,
-            Some(app_init_cfg),
-            base_path,
-        );
-        app.run()
-    }
 
-    /// Creates a new `AppBuilder` instance. It's the standard way to create a Seed app.
-    ///
-    /// Then you can call optional builder methods like `routes` or `sink`.
-    /// And you have to call method `build_and_start` to build and run a new `App` instance.
-    ///
-    /// _NOTE:_ If your `Model` doesn't implement `Default`, you have to call builder method `after_mount`.
-    ///
-    /// # Example
-    ///
-    /// ```rust,no_run
-    ///fn update(msg: Msg, model: &mut Model, _orders: &mut impl Orders<Msg, GMsg>) {
-    ///   match msg {
-    ///       Msg::Clicked => model.clicks += 1,
-    ///   }
-    ///}
-    ///
-    ///fn view(model: &Model) -> impl IntoNodes<Msg> {
-    ///   vec![
-    ///       button![
-    ///           format!("Clicked: {}", model.clicks),
-    ///           simple_ev(Ev::Click, Msg::Clicked),
-    ///       ],
-    ///   ]
-    ///}
-    ///
-    ///App::builder(update, view)
-    /// ```
-    pub fn builder(
-        update: UpdateFn<Ms, Mdl, INodes, GMs>,
-        view: ViewFn<Mdl, INodes>,
-    ) -> AppBuilder<Ms, Mdl, INodes, GMs, UndefinedInitAPI> {
-        // @TODO: Remove as soon as Webkit is fixed and older browsers are no longer in use.
-        // https://github.com/David-OConnor/seed/issues/241
-        // https://bugs.webkit.org/show_bug.cgi?id=202881
-        let _ = util::document().query_selector("html");
+        app.data
+            // @TODO_B we are trying to get rid of this field
+            .main_el_vdom
+            // TODO_B with the removal of MountType, this becomes argument-free. This needs looking at
+            .replace(Some(app.bootstrap_vdom()));
 
-        // Allows panic messages to output to the browser console.error.
-        console_error_panic_hook::set_once();
+        let mut orders = OrdersContainer::new(app.clone());
+
+        let new_model = init(
+            Url::current().skip_base_path(&Rc::clone(&app.cfg.base_path)),
+            &mut orders,
+        );
+        app.data.model.replace(Some(new_model));
+
+        routing::setup_popstate_listener(
```

Comment:
`routing::...` Moved up from `App::run`.  Se line 633 of old version. Only change is removal of function arguments.